licenses.yaml

```diff
@@ -147,6 +147,16 @@ source_paths:
 
 ---
 
+name: AWS RDS SDK for Java
+license_category: source
+module: extensions/druid-aws-rds-extensions
+license_name: Apache License version 2.0
+version: 1.11.199
+libraries:
+  - com.amazonaws: aws-java-sdk-rds
```

Comment:
Hi @himanshug, did you adopt source code from the AWS SDK? If so, `source_paths` field should be added. Otherwise, `license_category` should be `binary`.

Reply:
source code is not copied here , I guess this should be "binary" , yes. .. will update.